leaf-ui/css/analysis_html.css

```diff
@@ -17,6 +17,13 @@ body {
   font-family: Arial;
 }
 
+body.waiting,
```

Comment:
I'm not sure if you need the body.waiting, body.waiting a:hover since you have body.waiting * but I am not a css whiz and could be totally off base so feel free to correct me!

Reply:
I think you're right!! just changed it